### PR TITLE
Migrating unique BigNumber(/Total) controls

### PR DESCRIPTION
**Pull request by rusackas**
### CATEGORY

Choose one

- [ ] Bug Fix
- [ ] Enhancement (new features, refinement)
- [x] Refactor
- [ ] Add tests
- [ ] Build / Development Environment
- [ ] Documentation

### SUMMARY

Moving unique control(s) from `controls.jsx` to `/explore/controlPanels/BigNumber.js` and `/explore/controlPanels/BigNumberTotal.js`

### ADDITIONAL INFORMATION

- [x] Has associated issue: #9187
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### REVIEWERS
@kristw

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1016,15 +1016,6 @@ The `metric` (or equivalent) and `timeseries_limit_metric` fields are all compos
...
 
-### Options
-
-| Field                  | Type      | Notes                                |
-| ---------------------- | --------- | ------------------------------------ |
-| `compare_lag`          | _number_  | The **Comparison Period Lag** widget |
-| `compare_suffix`       | _string_  | The **Comparison suffix** widget     |
-| `show_trend_line`      | _boolean_ | The **Show Trend Line** widget       |
-| `start_y_axis_at_zero` | _boolean_ | The **Start y-axis at 0** widget     |
-
```
Comment (rusackas) [inline CONTRIBUTING.md]: @kristw I'll just keep removing these things as the controls are moved out of `controls.js`, but let me know if you have thoughts on when/where NOT to do that, so I don't kill any useful documentation.

Comment (kristw) [reply CONTRIBUTING.md]: I am not 100% sure about this part but seems reasonable I guess. Or you can copy the note as comment in the js file.

**Review by kristw — APPROVED** [review]

**Review by kristw — APPROVED** [review]

**Comment by ktmud** [comment]
I also made change to the Big Number controls here: https://github.com/apache/incubator-superset/pull/9341

It seems there will be conflicts. Which one should we merge first?

**Comment by kristw** [comment]
I'll merge the other one first then since this one's ci is still broken.

**Comment by rusackas** [comment]
Not really sure why CI is failing on this one... I just restarted it to see if I get lucky ;)

Happy to rebase (or even re-do) these changes on an updated `master`

**Comment by rusackas** [comment]
Closing in favor of #9440

**Pull request closed without merging**